index.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   linkBankOperations: require('./libs/linkBankOperations'),
   addData: require('./libs/addData'),
   filterData: require('./libs/filterData'),
+  updateOrCreate: require('./libs/updateOrCreate'),
   request: require('./libs/request'),
```

Comment:
I find the object oriented approach I took in cozy-konnector-linxo easier to understand. You have documents, they have fields that are used for unicity and then you can forget it, you do not have to call `updateOrCreate` knowing which fields should be used. I would be in favor of porting the object oriented Document class inside cozy-konnector-libs.

I think it would also help to reorganize addData/filterData/updateOrCreate into the same file/class.

Reply:
https://gitlab.cozycloud.cc/labs/cozy-konnector-linxo/blob/master/helpers/cozy.js

Reply:
I get you point, this won't be a part of this PR since I use to port the MAIF connector now.